routes/voice-alexa.js

```diff
@@ -844,9 +847,7 @@ function createQuestionTemplate(
 	let templateText = `${bodyText}<br/>`;
 	let builder, fontSize;
 	const includesAnswerContent = articleImage != null;
-
 	if (includesAnswerContent) {
```

Comment:
Why are we not falling back to the FT logo anymore?
There are still some articles that don't have images.